### ISSUE TITLE
Support for secure websockets?

### ISSUE THREAD

**Issue by J-Rojas** [fkie/async_web_server_cpp#2]
Hi, I have recently used the webrtc_ros node within a prototype. For my use case, I need a secure web context to workaround webrtc browser issues when not solely working through localhost.

It seems this project doesn't current support a configuration for secure websockets, is that correct?

**Comment by roehling** [comment]
No, but you can use pretty much any TLS proxy such as `stunnel` as frontend. For instance, I used the following configuration in the past to have a secure WebRTC server on port 8490 with the regular WebRTC server at localhost:8080 as backend:

    pid =
    foreground = yes

    [webrtc]
    accept = 8490
    connect = 8080
    cert = webrtc_stunnel_fake_cert.pem
    key = webrtc_stunnel_fake_key.pem

With `foreground = yes`, you can even start and stop the `stunnel` program via launch file.
